### PR TITLE
Fix for jena 5 upgrade

### PR DESCRIPTION
**Pull request by brinxmat**
- This needs extremely attentive testing since I trust the automated tests on expansion about zero percent
- It is the first step towards using the new expansion

### CODE REVIEW

File: expansion/src/main/java/no/unit/nva/expansion/utils/JsonLdDefaults.java [hunk 1]
```diff
@@ -1,39 +1,51 @@
...
+        RDFDataMgr.write(outputStream, model, RDFFormat.JSONLD);
+        try (var inputStream = new ByteArrayInputStream(outputStream.toByteArray())) {
+            var jsonDocument = JsonDocument.of(MediaType.JSON_LD, inputStream);
+            return write(JsonLd.frame(jsonDocument, frame).options(getJsonLdOptions()).get());
+        } catch (IOException | JsonLdError e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static String write(JsonObject frame) {
```
Comment (monaullah) [inline expansion/src/main/java/no/unit/nva/expansion/utils/JsonLdDefaults.java:36]: `JsonObject framedObject`?

File: schema-org-metadata/src/main/java/no/unit/nva/schemaorg/SchemaOrgDocument.java [hunk 2]
```diff
@@ -68,30 +73,33 @@ private static InputStream toInputStream(PublicationResponse publication) {
...
-                .build()
-                .asString();
-    }
-
-    private static JsonLDWriteContext getJsonLdWriteContext(String type) {
-        var context = new JsonLDWriteContext();
-        context.setOptions(getJsonLdOptions());
-        context.setFrame(createFrame(type));
-        return context;
+        var byteArrayOutputStream = new ByteArrayOutputStream();
```
Comment (monaullah) [inline schema-org-metadata/src/main/java/no/unit/nva/schemaorg/SchemaOrgDocument.java:76]: Very similar code/duplicate of code in `FramedJsonGenerator` - extract to common util?

Comment (brinxmat) [reply schema-org-metadata/src/main/java/no/unit/nva/schemaorg/SchemaOrgDocument.java:76]: Different dependencies make this a job for a new PR

**Review by truhacevkir — APPROVED** [review]
We can test it in sandbox before merging maybe?

**Review by monaullah — APPROVED** [review]
